### PR TITLE
Update in e2e tests to fix some minor bugs and image reference

### PR DESCRIPTION
**Pull request by lmzuccarelli**
This again is draft PR to check the gcp-e2e test works

### CODE REVIEW

File: bundle.Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=node-observability-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.18.0+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0+git
```
Comment (josefkarasek) [inline bundle.Dockerfile:9]: why down?

File: bundle/manifests/node-observability-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -311,7 +331,8 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/openshift/node-observability-operator:v0.0.1
+                image: quay.io/luzuccar/node-observability-operator:v1.0.0
```
Comment (josefkarasek) [inline bundle/manifests/node-observability-operator.clusterserviceversion.yaml:334]: private fork

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -25,7 +25,9 @@ spec:
         - /manager
         args:
         - --leader-elect
-        image: controller:latest
+        - --zap-log-level=debug
+        image: quay.io/luzuccar/node-observability-operator:v1.0.0
```
Comment (josefkarasek) [inline config/manager/manager.yaml:29]: private fork

File: pkg/operator/controller/nodeobservability/controller.go [hunk 4]
```diff
@@ -177,36 +176,39 @@ func (r *NodeObservabilityReconciler) Reconcile(ctx context.Context, req ctrl.Re
...
 	listOpts := []client.ListOption{
 		client.InNamespace(nodeObs.Namespace),
 		client.MatchingLabels(labelsForNodeObservability(nodeObs.Name)),
 	}
+
+	// list pods to ensure that the agents get deployed
 	if err = r.List(ctx, podList, listOpts...); err != nil {
 		r.Log.Error(err, "Failed to list pods", "NodeObservability.Namespace", nodeObs.Namespace, "NodeObservability.Name", nodeObs.Name)
-		return ctrl.Result{}, err
+		return reconcile.Result{}, err
```
Comment (josefkarasek) [inline pkg/operator/controller/nodeobservability/controller.go:188]: Why "sigs.k8s.io/controller-runtime/pkg/reconcile"? Yes, both structs are identical, but we're now mixing reconcile and controller-runtime. Func signature vs return values.

"sigs.k8s.io/controller-runtime"

**Review by josefkarasek — CHANGES_REQUESTED** [review]

File: bundle.Dockerfile (hunk 1 above)
Comment (lmzuccarelli) [reply bundle.Dockerfile:9]: Updated to newer version of opertaor-sdk - fixed

**Review by bharath-b-rh — COMMENTED** [review]
/lgtm

**Review by bharath-b-rh — APPROVED** [review]

**Pull request merged**